tests/Gaufrette/Functional/Adapter/AzureMultiContainerBlobStorageTest.php

```diff
@@ -108,6 +108,21 @@ public function shouldGetSize()
         $this->assertEquals($contentSize, $this->filesystem->size($path));
     }
 
+    /**
+     * @test
+     * @group functional
+     */
+    public function shouldGetMd5Hash()
+    {
+        $path = $this->createUniqueContainerName('container') . '/foo';
+
+        $content = 'Some content';
+        $this->filesystem->write($path, $content);
+        $checksumBase64 = $this->filesystem->checksum($path);
+
+        $this->assertEquals(\md5($content), \bin2hex(\base64_decode($checksumBase64, true)));
```

Comment:
I'm not sure to get your test here. Why to call again `bin2hex` and `base64_decode` ?
Shouldn't the test be
```php
$checksum = $this->filesystem->checksum($path);
$this->assertEquals(\md5($content), $checksum);
```
?

Reply:
Yup, you are right. It was too quick with this. Sending fix right away.